### PR TITLE
[macOS] Update homebrew formula

### PR DESCRIPTION
**Pull request by rabarar**
modified homebrew formula to support building with sopaysdr support by adding the depends as well as the -DSOAPYSDR=TRUE cmake flags

### CODE REVIEW

File: HomebrewFormula/welle.io.rb [hunk 1]
```diff
@@ -9,11 +9,12 @@ class WelleIo < Formula
   depends_on "qt"
   depends_on "fftw"
   depends_on "faad2"
-  depends_on "librtlsdr"
+  depends_on "soapysdr"
+  depends_on "soapyuhd"
```
Comment (mpbraendli) [inline HomebrewFormula/welle.io.rb]: Are there other soapy device drivers you could optionally depend on? What comes to mind: hackrf, limesdr, plutosdr, maybe others

**Comment by rabarar** [comment]
if you can tell me how you manage the version string, I’ll make the change and resubmit the PR.

**Comment by AlbrechtL** [comment]
> if you can tell me how you manage the version string

What do you mean with version string?
You can find the releases here: https://github.com/AlbrechtL/welle.io/releases
The link to the source code of 2.0 beta1 is: https://github.com/AlbrechtL/welle.io/archive/v2.0-beta1.tar.gz

**Comment by rabarar** [comment]
Misunderstood - I modified the url and the hash…

**Comment by AlbrechtL** [comment]
Thanks!

**Comment by rabarar** [comment]
😀

**Pull request merged**